docs/contributor/decision-records.md

```diff
@@ -0,0 +1,42 @@
+# Decision Records
+
+The following sections describe the main design decisions made.
+
+## Handling of Rate Limiting Calling IAS API
+
+We didn't implement any rate limit handling because the [Rate Limiting documentation of IAS](https://help.sap.com/docs/IDENTITY_AUTHENTICATION/6d6d63354d1242d185ab4830fc04feb1/e22ee47abf614565bcb29bb4ddbbf209.html) mentions the following: 
+
+> To ensure a safe and stable environment, all requests have a limit of 50 concurrent requests per second. The requests are associated with the originating IP address, and not with the user making the requests.
+
+Currently, we do not expect to exceed this rate limit as a reconciliation can perform a maximum of 5 sequential requests.  
+There is also mention of a specific rate limit for SCIM endpoints, but we do not use these endpoints.
+
+### Caching of Well-Known Token Endpoint
+
+We read the known configuration of the IAS tenant that is used to create the applications to obtain the token endpoint. This token endpoint is then stored in the secret on the managed runtime along with the client ID and the client secret.
```

Comment:
Shouldn't all the 'secrets' be capitalized?

Reply:
I will double-check with @ijovovic.